content/fr/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -65,51 +65,51 @@ Les EndpointSlices supportent 3 types d'addresses:
 
 * IPv4
 * IPv6
-* FQDN (Fully Qualified Domain Name) - [serveur entièrement nommé]
+* FQDN (Fully Qualified Domain Name) - [nom de domaine entièrement qualifié]
 
 ### Topologie
 
 Chaque Endpoint dans un EnpointSlice peut contenir des informations de topologie pertinentes. 
```

Comment:
EnpointSlice => EndpointSlice

Reply:
This has now been fixed with last commit. Thanks